NETProvider/src/FirebirdSql.Data.FirebirdClient/Client/Native/IFbClient.cs

```diff
@@ -269,4 +351,98 @@ public interface IFbClient
 
 		#endregion
 	}
+
+	internal static class IFbClientEx
+	{
+		public static IntPtr isc_dsql_allocate_statement_ex(
```

Comment:
If i.e. the `dbHandle` will be `FirebirdHandle` it will call this method. For `int` it would pass the call to native as it's now.